CHANGELOG.md

```diff
@@ -3,88 +3,112 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
-## 6.0.1-alpha.0 (2021-11-16)
-
-**Note:** Version bump only for package kongponents
-
-
 
-
-
-## [6.1.0](https://github.com/Kong/kongponents/compare/v5.0.8...v6.1.0) (2021-11-16)
+## [6.1.2](https://github.com/Kong/kongponents/compare/v5.0.8...v6.1.2) (2021-11-16)
 
 ### New Dependencies Added
 
 * `@vue/composition-api` and `swrv` added as `dependencies` for state management in table and card catalog
 * `vue-uuid` package added as `devDependency` for accessibility/ARIA standards
 * `tailwindcss` and `@kawarimidoll/vuepress-plugin-tailwind` added as `devDependencies` for utility classes
 
-### Breaking Changes
+### BREAKING CHANGES
 * `KAlert` 
-  - new props `type` and `dismissType` added (`is-dismissable` prop removed)
-  - `size` prop now supports `large` and `small` 
-  - new slots associated with "banner" `type` and `size` `large` (icon, description, actionButtons)
+  * `is-dismissable` prop removed
 
 * `KBadge` 
-  - add `info` option for `appearance` prop
-  - add `shape` property to support rectangular badges
+  * look and feel changed to match redesign
 
 * `KButton` 
-  - `size` prop now supports `small`, `medium`, and `large` 
-  - `isRounded` property added and defaults to `true`
-  - Buttons default to a rounded appearance
+  * Buttons default to a rounded appearance
 
 * `KCard` 
-  - `helpText` property removed, 
-  - `status` / `statusHat` prop and slot added 
-  - `notifications` slot added
+  * `helpText` property removed
+  * changes to look and feel
+
+* `KCheckbox` 
+  * `label` property removed
+
+* `KIcon` 
+  * changed the way SVGs are rendered
```

Comment:
Add more detail, something like:

> SVGs are now wrapped with `<span>` and `KIcon`-specific classes are now applied to the wrapper.

I think pointing out the SVG is now not the tag to target here is important since so many styles changed as a result.

Reply:
Done!